src/components/about/about.jsx

```diff
@@ -4,10 +4,10 @@ const About = () => {
   return (
     <div className="bg-witl-purple-300 min-h-screen flex flex-col rounded-b-[48px] items-center justify-center mt-[5%]">
       <div className="flex flex-col tracking-widest">
-        <div className="mb-1 text-9xl text-center text-witl-purple-100">
+        <div className="mb-1 text-9xl max-sm:text-8xl text-center text-witl-purple-100">
```

Comment:
let's make it a bit smaller since it looks like we're trying to just barely smash in "about" in the screen.

also, it should be something like `text-7xl md:text-9xl` instead of `text-9xl max-sm:text-8xl`

what text-7xl md:text-9xl basically does is state that the **about** text will always be text-7xl unless the screen's size crosses the breakpoint of md (medium screens), and once it crosses that breakpoint, the size will always be text-9xl